west.yml

```diff
@@ -18,9 +18,12 @@ manifest:
   defaults:
     remote: upstream
 
+
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name:  nxp_dev
+      url-base: https://github.com/hakehuang/ 
```

Comment:
change this before merge

Reply:
ok, I am discussion with maureen on how to create pull request for nxp_hal, once merged I will change align with mainline